### ISSUE TITLE
Ability to export / import users list

### ISSUE THREAD

**Issue by brunobord** [vole/vole#53]
Use cases:
- I want to swap from a machine to another, and that I want to keep up to date with the people I'm following,
- My disk is crashing, 

I wish I could backup at least my user list, to put it on a USB disk, and recreate the whole user list in a snap.

**Comment by sbruce** [comment]
You could have your config/users list be in it's own folder that could be shared using BTsync. You wouldn't give this folder key to your friends, but it could be used to backup your information. If you wanted Vole installed on a second machine, you could enter the R/W key of your config folder, and the two machines would sync that folder with each other.

**Comment by marktheunissen** [comment]
Yup, we'll need something like this for sure. Hopefully it will be easy with a BitTorrent API to read the directory and ensure that the correct users are followed. I like the idea of having a separate share for private data that keeps the app in sync.

The other option is to have a hosted Vole instance - either self host or SaaS. Then you always access the same web frontend - authentication will be needed, and of course this doesn't suit everyone but it is possible. It's also the way we could support mobile easily.
